benchmarks_gui/CMakeLists.txt

```diff
@@ -41,13 +51,17 @@ include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 link_directories(${Boost_LIBRARY_DIRS})
 link_directories(${catkin_LIBRARY_DIRS})
 
-qt4_wrap_cpp(MOC_FILES include/main_window.h include/trajectory.h include/benchmark_processing_thread.h)
-qt4_wrap_ui(UIC_FILES
-  src/ui/main_window.ui
+set(MOC_FILES include/main_window.h include/trajectory.h include/benchmark_processing_thread.h)
+set(UI_FILES src/ui/main_window.ui
   src/ui/run_benchmark_dialog.ui
   src/ui/robot_loader.ui
   src/ui/bounding_box_goals.ui
 )
+if (UseQt5)
+qt5_wrap_ui(UIC_FILES ${UI_FILES})
+else (UseQt5)
```

Comment:
For example, in this random project they have a regular else:
http://sourceforge.net/u/brankec69/fleetingpm/ci/873cc5ef45634e0b02e17a7445b0d216c22905f0/tree/CMakeLists.txt